policy-report/image-vuln-adapter/pkg/kube/object.go

```diff
@@ -0,0 +1,249 @@
+package kube
```

Comment:
Is there a way we can reuse this from Starboard instead of duplicating all of this?

Reply:
will work on it.